libraries/chain/include/graphene/chain/asset_object.hpp

```diff
@@ -184,6 +223,11 @@ namespace graphene { namespace chain {
          /// The tunable options for BitAssets are stored in this field.
          bitasset_options options;
 
+         /// Check collateral-denominated fees:
+         template<class DB>
+         bool collateral_fees_are_zero(const DB& db) const
+         { return asset_id(db).dynamic_asset_data_id(db).accumulated_collateral_fees > 0; }
```

Comment:
The function name `are_zero` is inconsistent with the check `>0`.

BTW it seems the function is unused.

Reply:
Thanks.  Removed method in 02f9da8.  Had anticipated using in evaluation checks for asset change of backing collateral, but seemed cleaner just to address the field directly.